src/languages/java.md

```diff
@@ -1,3 +1,35 @@
 # Java
 
-Java support is in Alpha and not generally available. Contact sales [https://platform.sh/contact](https://platform.sh/contact) if you are interested in running Java based applications on Platform.sh.
+Platform.sh supports building and deploying applications written in Java. It has built-in support for Maven, Ant and Gradle.
+
+## Supported versions
+
+We support both the Oracle JVM and the Open JDK but we strongly suggest you use the Open JDK as support for the Oracle JVM may be dropped in the future if they continue with their licensing shenanigans.
```

Comment:
That's really not the way we should phrase it.  It may be accurate but it's kinda petulant in the documentation.